solidity/test/integration/IntegrationBase.sol

```diff
@@ -6,30 +6,47 @@ import {Test} from 'forge-std/Test.sol';
 
 import {ConnextVestingWallet} from 'contracts/ConnextVestingWallet.sol';
 
+import {Deploy} from 'scripts/Deploy.sol';
 import {Constants} from 'test/utils/Constants.sol';
-import {ILlamaPay} from 'test/utils/ILlamaPay.sol';
-import {ILlamaPayFactory} from 'test/utils/ILlamaPayFactory.sol';
+
+import {IVestingEscrowSimple} from 'interfaces/IVestingEscrowSimple.sol';
+import {IVestingEscrowFactory} from 'test/utils/IVestingEscrowFactory.sol';
 
 // TODO: Inherit and run Deploy.sol script, instead of deploying the contract here
 contract IntegrationBase is Test, Constants {
-  address public owner = makeAddr('owner');
+  address public owner; // set later
   address public payer = makeAddr('payer');
 
   IERC20 internal _nextToken = IERC20(NEXT_TOKEN_ADDRESS);
-  ILlamaPayFactory internal _llamaPayFactory = ILlamaPayFactory(LLAMA_FACTORY_ADDRESS);
+  IVestingEscrowFactory internal _llamaVestFactory = IVestingEscrowFactory(LLAMA_FACTORY_ADDRESS);
 
   ConnextVestingWallet internal _connextVestingWallet;
-  ILlamaPay internal _llamaPay;
+  IVestingEscrowSimple internal _llamaVest;
 
   function setUp() public virtual {
     vm.createSelectFork(vm.rpcUrl('mainnet'), FORK_BLOCK);
 
-    _connextVestingWallet = new ConnextVestingWallet(owner, 24_960_000 ether);
-    //! TODO: Replace for LlamaPay V2
-    _llamaPay = ILlamaPay(_llamaPayFactory.createLlamaPayContract(NEXT_TOKEN_ADDRESS));
+    // deploy
+    Deploy _deploy = new Deploy();
+    _deploy.run();
+    owner = _deploy.OWNER();
+    _connextVestingWallet = _deploy.connextVestingWallet();
 
     deal(NEXT_TOKEN_ADDRESS, payer, TOTAL_AMOUNT);
+
+    // address _llamaVestAddress = ContractDeploymentAddress.addressFrom(address(_llamaVestFactory), 0);
+    address _llamaVestAddress = 0xB93427b83573C8F27a08A909045c3e809610411a; // hardcoded from logs
```

Comment:
not following, why cannot you create the llama vest contract instead of hardcoding this?
`VestFactory.create`?